calling/src/commonMain/kotlin/com/wire/kalium/calling/CallType.kt

```diff
@@ -0,0 +1,7 @@
+package com.wire.kalium.calling
+
+enum class CallType(val value: Int) {
+    NORMAL(value = 0),
+    VIDEO(value = 1),
+    FORCED_AUDIO(value = 2)
+}
```

Comment:
```suggestion
enum class CallType(internal val avsValue: Int) {
    NORMAL(avsValue= 0),
    VIDEO(avsValue= 1),
    FORCED_AUDIO(avsValue= 2)
}
```

Also, I suggest adding some Kdocs on this. What's the difference between `FORCED_AUDIO` and `NORMAL`, for example?
So we don't need to go through AVS source code to understand what exactly they mean.